tools/Customization/DevHome.FileExplorerSourceControlIntegration/Services/RepositoryTracking.cs

```diff
@@ -146,4 +149,15 @@ public string GetSourceControlProviderForRootPath(string rootPath)
             }
         }
     }
+
+    public void ReloadRepositoryStoreIfChangesDetected()
+    {
+        var lastTimeModified = System.IO.File.GetLastWriteTime(Path.Combine(RepoStoreOptions.RepoStoreFolderPath, RepoStoreOptions.RepoStoreFileName));
+        log.Information("Last Time Modified: {0}", lastTimeModified);
+        if (DateTime.Compare(LastRestore, lastTimeModified) < 0)

```

Comment:
Consider changing both `LastRestore` and `lastTimemodified` to UTCtime for comparison.  This can fail during daylight savings time.

A specific edge case, but can still happen.